contracts/Aavegotchi/facets/ItemsFacet.sol

```diff
@@ -255,121 +258,65 @@ contract ItemsFacet is Modifiers {
                 delete aavegotchi.equippedWearables[slot];
 
                 // remove wearable from Aavegotchi and transfer item to owner
-                _removeWearableFromGotchi(_tokenId, existingEquippedWearableId, slot, _gotchiInfo);
+                if (_gotchiInfo.equippedDepositIds[slot] != 0) {
+                    // remove wearable from Aavegotchi and delete delegation
```

Comment:
Wearable continues to be delegated to that user. It only "deletes delegation" when the role is revoked 
```suggestion
                    // remove wearable from Aavegotchi
```